integration-tests/utils/workflow-utils.js

```diff
@@ -22,13 +22,18 @@ const launchWorkflowAndWaitForSuccess = async page => {
 
   // Long enough for the submission details to be Running or Failed.
   // Workflows might sit in "Submitted" for a long time if there is a large backlog of workflows in the environment
-  const isRunning = await Promise.race([
-    findInGrid(page, 'Failed', { timeout: 5 * 60 * 1000 }).then(() => false),
-    findInGrid(page, 'Running', { timeout: 5 * 60 * 1000 }).then(() => true)
+  const workflowStatus = await Promise.race([
+    findInGrid(page, 'Succeeded', { timeout: 5 * 60 * 1000 }).then(() => 'Succeeded'),
+    findInGrid(page, 'Failed', { timeout: 5 * 60 * 1000 }).then(() => 'Failed'),
+    findInGrid(page, 'Running', { timeout: 5 * 60 * 1000 }).then(() => 'Running')
```

Comment:
I admittedly do not fully understand how this test works, but it seems like we could completely ignore `Running`?

Testing the outcome of `Succeeded` transitively verifies the implementation detail that it was `Running` at some point in the past, and tests are best when they are about outcomes.

```suggestion
  const workflowStatus = await Promise.race([
    findInGrid(page, 'Succeeded', { timeout: 5 * 60 * 1000 }).then(() => 'Succeeded'),
    findInGrid(page, 'Failed', { timeout: 5 * 60 * 1000 }).then(() => 'Failed'),
```

Reply:
I'm going to go ahead and merge this as-is to attempt to resolve the current flakiness in tests. I agree with what you're saying though. Does anyone in the Batch or QA teams want to take on further updates to this test?